exercises/practice/simple-linked-list/SimpleLinkedListTests.cs

```diff
@@ -41,8 +41,8 @@ public void Two_item_list_second_item_has_no_next()
     [Fact(Skip = "Remove this Skip property to run this test")]
     public void Implements_enumerable()
     {
-        var values = new SimpleLinkedList<int>(2).Add(1);
-        Assert.Equal(new[] { 2, 1 }, values);
+        var values = new SimpleLinkedList<int>(2).Add(1).Add(3);
+        Assert.Equal(new[] { 2, 1, 3 }, values);
```

Comment:
I like the idea of using `Assert.IsAssignableFrom`. A number of tests could be improve like that. 
I still think that having three values in the list allows for a more robust test. 

The testing of reverse connects to the bigger discussion about whether this list should be LIFO or FIFO. Reverse doesn't have to be directly implemented as it is provided by [Enumerable](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.reverse?view=net-8.0). Testing it can help catch cases where somebody implements `IEnumerable` without using the base class. In other languages reversing a LIFO is an interesting problem and a classic algorithm to learn, but we cannot do that here as we don't ask for the `Pop` or `Remove` operation to be implemented. This links to [Problem 2 in the forum post](https://forum.exercism.org/t/incorrect-solutions-passing-tests-in-c-track/4840/8#problem-2-dissimilarity-between-instructions-and-tests-2) and also the [something else to consider](https://forum.exercism.org/t/incorrect-solutions-passing-tests-in-c-track/4840/8#something-else-to-consider-4) section.

Reply:
> I still think that having three values in the list allows for a more robust test.

Yeah sure! I hadn't even noticed that.

Reply:
I've left some comments on the forum post